rpm/389-ds-base.spec.in

```diff
@@ -559,6 +566,28 @@ fi
 %systemd_postun_with_restart %{pkgname}-snmp.service
 
 
+%post entryuuid
+# Add the entryuuid plugins
+instbase="%{_sysconfdir}/%{pkgname}"
+ninst=0
+for dir in $instbase/slapd-* ; do
+    if [ ! -d "$dir" ] ; then continue ; fi
```

Comment:
I'm a bit worried that this may just install into instances that people aren't expecting it to be added to. I think the postun is fine, but you shouldn't install into every instance by default

Reply:
I agree, I think it should be a conscious decision of an admin to install and enable the plugin, as it might impact replication with older systems.

Reply:
Like I mentioned to Vitkor.  The current/previous behavior is that we DO add it to all instances by default and turn it on.  That's what your update code does. :-)